src/main/resources/application.yml

```diff
@@ -12,6 +12,7 @@ spring:
     url: "jdbc:postgresql://${database.endpoint:localhost:5432}/${database.name:postgres}"
     user: "${database.username:root}"
     password: "${database.password:dev}"
+    mixed: true
```

Comment:
Not sure about this - it allows non-TX to be included in otherwise transactional schema updates. I know it's needed here but it applies to every update?!

To get round it would just need 2 files instead of 1 (I think)